lambdas/postgres-migration-count-tool/main.tf

```diff
@@ -14,20 +14,24 @@ resource "aws_lambda_function" "postgres_migration_count_tool" {
 
   environment {
     variables = {
-      databaseCredentialSecretArn = var.rds_user_access_secret_arn
-      ExecutionsTable             = var.dynamo_tables.executions.name
-      GranulesTable               = var.dynamo_tables.granules.name
-      PdrsTable                   = var.dynamo_tables.pdrs.name
-      dbHeartBeat                 = var.rds_connection_heartbeat
+      acquireTimeoutMillis        = var.rds_connection_timing_configuration.acquireTimeoutMillis
       AsyncOperationsTable        = var.dynamo_tables.async_operations.name
       CollectionsTable            = var.dynamo_tables.collections.name
+      createRetryIntervalMillis   = var.rds_connection_timing_configuration.createRetryIntervalMillis
+      createTimeoutMillis         = var.rds_connection_timing_configuration.createTimeoutMillis
+      databaseCredentialSecretArn = var.rds_user_access_secret_arn
       ExecutionsTable             = var.dynamo_tables.executions.name
+      ExecutionsTable             = var.dynamo_tables.executions.name
+      GranulesTable               = var.dynamo_tables.granules.name
       GranulesTable               = var.dynamo_tables.granules.name
+      idleTimeoutMillis           = var.rds_connection_timing_configuration.idleTimeoutMillis
+      PdrsTable                   = var.dynamo_tables.pdrs.name
       PdrsTable                   = var.dynamo_tables.pdrs.name
```

Comment:
executions, granules and pdrs tables are duplicated here

Reply:
046aa014c